packages/devtools/devtools/src/initialize.tsx

```diff
@@ -18,30 +19,32 @@ import { PanelsContainer } from './containers';
 import { sections } from './sections';
 import { theme } from './theme';
 
-const Devtools = ({ clientReady }: { clientReady: Event<Client> }) => {
-  const [client, setClient] = useState<Client>();
+export type ClientAndServices = { client: Client; services: ClientServices }
+
+const Devtools = ({ clientReady }: { clientReady: Event<ClientAndServices> }) => {
+  const [value, setValue] = useState<ClientAndServices>();
 
   useEffect(() => {
-    clientReady.on((client) => setClient(client));
+    clientReady.on((value) => setValue(value));
   }, []);
 
   return (
     <ErrorBoundary>
       <ThemeProvider theme={theme}>
         <CssBaseline />
         <FullScreen>
-          <Loader loading={!client} label='Loading DXOS Client...' />
-          {client && (
-            <ClientProvider client={client}>
+          <Loader loading={!value} label='Loading DXOS Client...' />
+          {value && (
+            <ClientContext.Provider value={value}>
```

Comment:
Realizing that nothing inside `react-client` is actually setting the `services` value in the context...I think that either we want the `ClientProvider` to set that or we should make a separate context for it. Given the comment that `services` is specifically for devtools I think it'd be reasonable to pull that code out here.

@zhenyasav what do you think? The `useClientServices` and `useDevtools` hooks probably don't need to be a part of the sdk, do they?